### PR TITLE
Intro paragraphs

### PR DESCRIPTION
**Pull request by tpoisot**
- [x] interactions are variable + respond to the environment
- [ ] specific importance of disease (waiting on @Norma-Forero refs)

### CODE REVIEW

File: manuscript.md [hunk 1]
```diff
@@ -43,6 +43,8 @@ landscape modification [@Edwards2021TroLan; @Magioli2021DefLea;
 possible in the prediction of networks, both extant (imputation) and future
 (forecasting).
 
+**Paragraph on disease goes here**
```
Comment (Norma-Forero) [inline manuscript.md:46]: Understanding disease transmission requires different methodological approaches that deal with interaction complexity. Networks assessment of vector-borne diseases, which have several hosts and intricate transmission cycles, is one of the examples in the field of disease ecology research. Few studies have implemented machine learning techniques for forecasting the incident cases of dengue, Chagas disease, rickettsiosis, leishmaniasis, as well as livestock and wildlife infectious diseases. 
  (Trop. Med. Infect. Dis. 2020, 5, 64; doi:10.3390/tropicalmed5020064 ; Phil. Trans. R. Soc. B 370: 20140107. http://dx.doi.org/10.1098/rstb.2014.0107  ;  PLOS Neglected Tropical Diseases | https://doi.org/10.1371/journal.pntd.0008056 September 24, 2020 ; Parasitology, Page 1 of 13. doi:10.1017/S0031182016002468  ; PLoS ONE 4(5): e5725. doi:10.1371/journal.pone.0005725 )

File: manuscript.md [hunk 2]
```diff
@@ -21,6 +21,30 @@ models that enable prediction (for the present) and forecasting (for the future)
...
+A comprehensive and general approach is required because of the tremendous
+diversity of species interaction types --- and how difficult they may be to
+predict. @Haas2019MitBee, for example, show that mites of bumblebee are
+responding more to host taxonomy than to geographical factors. In mutualistic
+interactions, different categories of species can show different degree of
+interaction intimacy [@Guimaraes2007IntInt]; recent work by @Hembry2018DoeBio
+reveals that more intimate interactions are inherently more stable (and could be
+easier to predict), which nicely complements results by @Trojelsgaard2015GeoVar,
+namely that partner fidelity among mutualists are responding to life history,
+geography, and other factors. Other types of interaction have proven to be
```
Comment (TanyaS08) [inline manuscript.md:33]: 'Other types of interaction _s_ ?

**Review by TanyaS08 — COMMENTED** [review]
Looks good, I think it works here place meant-wise but I do wonder if it has a knock-on effect on the 'what are interactions really' section? It probably doesn't but just wanted to raise/flag the thought

**Review by TanyaS08 — APPROVED** [review]

**Review by gottacatchenall — APPROVED** [review]

**Comment by gottacatchenall** [comment]
i made some edits to condense

**Comment by tpoisot** [comment]
I added a ref, and I think we're good to go with this PR

**Review by gottacatchenall — APPROVED** [review]

**Pull request merged**